mtags/src/main/scala-2/scala/meta/internal/pc/ScalaPresentationCompiler.scala

```diff
@@ -578,6 +597,7 @@ case class ScalaPresentationCompiler(
     val vd = new VirtualDirectory("(memory)", None)
     val settings = new Settings
     settings.Ymacroexpand.value = "discard"
+    settings.YpresentationDelay.value = 500
```

Comment:
Why is this option needed? Will this not cause the other features to become sluggish as well?